### DISCUSSION TITLE
Error updating diagram in draw.io for Notion

### DISCUSSION THREAD

**Discussion by ghost** [jgraph/drawio#3843] [Q&A]
When I edit a diagram in draw.io for Notion and complete it, the following error appears.

Error Message:
www.notion.so says
Error 400: Cannot update diagram.

I have tried several browsers and get the same error.

**Comment by walteriousnut** [comment]
@Toshiki-Higa is your Error 400: or 403? We receive this same error but is Error 403: Cannot update diagram

We have the same issue as of very recently. Cannot add new diagrams via the browser extension either. Editing existing diagrams successfully opens the diagram in the editor but fails with "Error 403: Cannot update diagram" when attempting to save the diagram changes.
